quickstarts/cockroach-db/config.yml

```
@@ -0,0 +1,48 @@
+id: e2ee8b6e-5c2a-11ed-9b6a-0242ac120002
+slug: cockroach database-newrelic
```

The slug can only have letters, numbers, and hyphens. Perhaps we should use this instead:

```suggestion
slug: cockroach-database-newrelic
```